benchmarks/profile_conversion_time.py

```diff
@@ -0,0 +1,101 @@
+# coding: utf-8
```

Comment:
We want to limit top level directories - can we stick this under tools?

Reply:
Done.